akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonManager.scala

```diff
@@ -626,6 +626,10 @@ class ClusterSingletonManager(singletonProps: Props, terminationMessage: Any, se
         goto(BecomingOldest).using(BecomingOldestData(oldest.filterNot(_ == cluster.selfUniqueAddress)))
       else
         goto(Younger).using(YoungerData(oldest.filterNot(_ == cluster.selfUniqueAddress)))
+
+    case Event(HandOverToMe, _) =>
+      // nothing to hand over in start
+      stay()
```

Comment:
Ok, so this is if we got a HandOverToMe before we saw that we joined the cluster and the buffer started and sent us `InitialOldestState`, which could lead to us being the oldest or not.

Could it make sense to stash the handover and unstash it as soon as we switch so that the HandOverToMe request is handled correctly as soon as possible?

Looking a bit more at the next step, in `BecomingOldest` the `HandOverToMe` is also not handled.

Reply:
Good hint, I'll look into that.

Reply:
Updated pull request with stash

Reply:
Reverted back to simple ignore, see comment by @patriknw in #28979